app/org/sagebionetworks/bridge/config/BridgeProductionSpringConfig.java

```diff
@@ -78,7 +94,7 @@ private JedisPool constructJedisPool(final String url, final JedisPoolConfig poo
         // With changes in Redis provisioning, passwords are now parseable by Java's URI class.
         URI redisURI = new URI(url);
         String password = redisURI.getUserInfo().split(":",2)[1];
-
+        
         if (bridgeConfig.isLocal() || password.equals("AWS")) {
             return new JedisPool(poolConfig, redisURI.getHost(), redisURI.getPort(),
                     bridgeConfig.getPropertyAsInt("redis.timeout"));
```

Comment:
As long as you're in here, you should clean up this logic, which will provide a fix for https://sagebionetworks.jira.com/browse/BRIDGE-1585 and remove the "AWS" hack. Should be pretty trivial to do.